controlpanel/frontend/jinja2/datasource-access-update.html

```diff
@@ -22,7 +22,7 @@ <h1 class="govuk-heading-xl">{{ page_title }}</h1>
   </form>
 </section>
 
-{% if request.user.has_perm('api.destroy_items3bucket', items3bucket) %}
+{% if request.user.has_perm('api.destroy_users3bucket', users3bucket) %}
```

Comment:
The context variable `items3bucket` is set in the view (in `controlpanel/frontend/views/datasource.py`), so this needs to remain `items3bucket`. The permission name change should work, though.